src/ucp/wireup/wireup.h

```diff
@@ -44,7 +44,12 @@ typedef struct ucp_wireup_msg {
     ucp_rsc_index_t               dst_aux_index;    /* Index of receiver wireup resource */
     uint16_t                      flags;            /* Wireup flags */
     uint8_t                       addr_len;         /* Length of first address */
-    /* addresses follow */
+    uint8_t                       peer_name_len;    /* Length of peer name */
```

Comment:
in case ENABLE_DEBUG_DATA is 0, this field still exits, but its always equal to 0. since this is one byte in connection request message, IMHO this is not significant.